app/user/rest.py

```diff
@@ -464,15 +464,12 @@ def send_contact_request(user_id):
         pass
 
     # Update the engagement stage in Salesforce for go live requests
-    if contact and contact.is_go_live_request and current_app.config["FF_SALESFORCE_CONTACT"]:
+    if contact and contact.is_go_live_request() and current_app.config["FF_SALESFORCE_CONTACT"]:
```

Comment:
Wow, that was the cause?! yikes

Reply:
haha, I know - @andrewleith came in clutch here